app/views/provider_interface/start_page/show.html.erb

```diff
@@ -40,7 +40,9 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-one-half">
         <h2 class="govuk-heading-m">View applications</h2>
-        <p class="govuk-body">See the status of applications at a glance.</p>
+        <p class="govuk-body">See newly submitted applications at a glance.</p>
+        <p class="govuk-body">Filter by status, training provider, ratifying body – and soon, reject by default date.</p>
+        <p class="govuk-body">Choose who in your organisation can make decisions and view sensitive info.</p>
```

Comment:
Is this premature? I think it might not be turned on for a while https://ukgovernmentdfe.slack.com/archives/CPH8J9G65/p1591786673129600

Reply:
Have put this text behind the relevant feature flag.

Reply:
I think we should use the `provider_add_provider_users` flag. It won't be turned on until all the others are.